phpBB/adm/style/permissions.js

```diff
@@ -302,3 +317,36 @@ function set_role_settings(role_id, target_id) {
 		mark_one_option(target_id, r, (settings[r] === 1) ? 'y' : 'n');
 	}
 }
+
+function match_role_settings(id)
+{
+	var fs = document.getElementById(id),
```

Comment:
I understand that these might be in line with some of the older JS but you should use variable names that actually mean something. These are not helpful at all and remind me a bit of really old C code. ;)